### ISSUE TITLE
origin: native range vs current establishment

### ISSUE THREAD

**Issue by peterdesmet** [qgroom/ias-dwc-proposal#5]
There seem to be two `origin` concepts going around in the IAS world:
## Origin as suggested in this repo

> To express the concept that a taxon has either been introduced by human activity or occurs naturally in a location and has done so before the impact of modern humans.

With controlled vocab:

```
native
reintroduced
introduced
introduced on or before 1500
introduced after 1500
vagrant
uncertain
```

This is what @timadriaens and @SoVDH have called `status` and has traditionally been populated in `establishmentMeans`. I think `status` is too generic of a name, but maybe `origin` is too. Would `establishment` be a good alternative? What have other IAS groups suggested, that could work in the larger scope of biodiversity information?
## Origin as native range

See https://github.com/inbo/alien-species-checklist/issues/53, where the term `origin` is currently proposed to indicate the native range, with values such as:

```
Africa
Europe
...
```

Maybe we need to just use `nativeRange` for this term, but I'm not sure if that is what is actually meant. Do we mean:

> where the species naturally occurs

Or:

> where the species came from, without making a claim about its status there (i.e. it could be introduced there as well)

@timadriaens and @SoVDH, what do you think?

**Comment by qgroom** [comment]
Yes, I've seen this. When I started writing the proposal I was going to suggest 'temporality' to indicate what I have now called origin, but I decided on the later term to make it consistent with the IUCN terms. Both origin and status are vague, but both are widely used. Nevertheless, I felt that a using a vague term may force people to read the definition and vocabulary before they use it. It is, in fact, quite difficult to choose a single word or short phrase to specifically convey these concepts.

**Comment by SoVDH** [comment]
We are currently working on this through the mapping exercise for the alien checklist. Especially the origin/status discussion 
Sorry guys but you are progressing too quicky for us ( speaking for me , maybe Tim too ?) Quite a hot and important topic actually and we need to do it properly.

**Comment by timadriaens** [comment]
If status is too general a term (which I agree, it can also indicate trend, rarity etc.), origin is too. For status, we could suggest renaming that term to "invasion stage" (in line with the Blackburn et al. 2014 framework). For the Belgian register, we decided to rename the field origin to "native range". And we use the NOBANIS terms for it:

Continents
Africa The African continent
Antarctica The Antarctic continent
Asia Divides Asia from Europe along Ural/Kazakhstan, Black Sea
Europe Divides Europe from Asia along Ural/Kazakhstan
N. America Canada, USA and Mexico
Oceania Australia, New Zealand, Tasmania and a number of small islands
S. America All countries on the S. American continent south of Mexico

Oceans
Arctic Ocean The ocean surrounding the North Pole
Indian Ocean The Ocean south of India to the southern tip of Africa and to the northern tip of Australia
N. Atlantic Ocean Upper part of the Atlantic Ocean divided by the equator
S. Atlantic Ocean The lower part of the Atlantic Ocean divided by the equator
N. Pacific Ocean The upper part of the Pacific Ocean divided by the equator
S. Pacific Ocean The lower part of the Pacific ocean divided by the equator
Southern Oceans The ocean surrounding the South Pole
combinations (also across continents | oceans)

field term = "native range", to discriminate between natural range expansion and a true alien is important and we don’t want to lose the information of where a species naturally occurs e.g. species where it is unknown whether they had natural range expansion or were introduced. Therefore, we can have EU in the area of origin (e.g. pontocaspian species, central european spp). We don’t follow the gisin terms here since these in our opinion better fit "status". There seems to be no corresponding controlled vocabulary in GRIIS for origin. The idea is to link origin with status and to distinguish primarily between species native to other parts of the EU from species from other continents.

**Comment by peterdesmet** [comment]
@timadriaens it seems you indeed want to indicate `where the species naturally occurs` (my first definition) and I would then indeed rename it to `nativeRange`.

**Comment by qgroom** [comment]
@SoVDH Don't worry about the speed, nothing is going to change in Darwin Core until at least the TDWG conference in December and may be not even then. The point is to get the discussion going, so that something will eventually be decided.

@timadriaens none of these terms is unambiguous or usable without clear definitions that mention the context. To my knowledge, Darwin Core is used to describe observations and create checklists, it is not used to describe species. Therefore you would never need to use "native range" in the context of which part of the world it comes from. You just need to be able say whether it comes from the area that the observation or checklist covers. In the proposal I was quite careful to make this distinction, because if you don't specify the context it is even more confusing.

I think 'origin' (as in the proposal) could be used in checklists and possibly with observations, but the latter use is more problematic. 'occurrenceStatus' is best used only with checklists and 'establishmentMeans' is best used with observations, but delimiter separated lists are allowed for some fields in DwC.

**Comment by timadriaens** [comment]
ok, makes perfect sense. I guess it's clear we have to do our mapping exercise first and then discuss with you what and how we can map certain fields on darwin core - or not.

**Comment by qgroom** [comment]
I've just been looking through Darwin Core and TDWG standards and it looks like there never has been a field for describing the biogeographic range of a species, though there may be one in ABCD, but I have not looked.

**Comment by qgroom** [comment]
@timadriaens What you are looking for is described by PlinianCore https://github.com/PlinianCore/Documentation/wiki

**Comment by peterdesmet** [comment]
While working on the [Manual of Alien plants](https://github.com/trias-project/alienplantsbelgium) I stumbled on the same problem as discussed here. There's a good argument for adopting the exact same name as in the IUCN definition, but as a term/[description type](https://github.com/trias-project/alienplantsbelgium/blob/27c894ad73b07894d08714eacec2af6486f7d415/data/processed/description.csv) name I find "origin" very broad.

To reiterate, the proposed vocab for the term is:

```
native
reintroduced
introduced
introduced before the modern era
introduced during the modern era
vagrant
unknown
```

Here are a number alternatives for the name `origin` (including bad ones and why they are bad):

* **speciesOrigin**: restricts scope of "origin", but "species" is not used in DwC terms.
* **taxonOrigin**: restricts scope of "origin", "taxon" is what is used in DwC terms, but maybe not clear this is about distribution.
* **distributionOrigin**: restricts scope of "origin", but kinda restricts its use to the distribution extension (which might be a good thing) and can be confused with native range.
* **speciesStatus**: could be anything and confused with name status (accepted, synonym)
* **invasionStage**: cf. [this vocab](https://github.com/inbo/alien-species-checklist/blob/master/data/vocabularies/invasionStage.tsv) suggested by @timadriaens and @SoVDH, but that one is about progression `established`, `invasive`, which is not want `native`, `introduced` is about.
* **nativeStatus**: one reason we're interested in this information is because we value native species (see the [justification in the proposal](https://github.com/qgroom/ias-dwc-proposal/blob/master/proposal.md#origin-new-term)). This term indicates that quite clearly. It also works well with `nativeRange` (used in the Manual) or other statuses like `threatStatus` (used in the [distribution extension](http://rs.gbif.org/extension/gbif/1.0/distribution.xml)) and `occurrenceStatus` (`present`, `absent`, see [this vocab](https://github.com/qgroom/ias-dwc-proposal/blob/master/vocabulary/presence.tsv)).

What do you think?
